### PR TITLE
GitHub actions added

### PR DESCRIPTION
**Pull request by volovyk-s**

### CODE REVIEW

File: commands/dev-deploy.js [hunk 1]
```diff
@@ -85,6 +84,10 @@ async function devDeploy(options) {
     // Deploy with init function and args
     const actions = [transactions.deployContract(readFileSync(options.wasmFile))];
 
+    if (options.initArgs && !options.initFunction) {
+        options.initFunction = 'new';
```
Comment (ailisp) [inline commands/dev-deploy.js:88]: Why we use this over `default: 'new'` ?

**Review by ailisp — APPROVED** [review]
CI setup looks good. The eslint danger is failing, not sure if that need to be fixed

**Comment by volovyk-s** [comment]
@ailisp I will fix danger in a separate PR. Or just set API keys, looks like this is the problem.

File: commands/dev-deploy.js (hunk 1 above)
Comment (volovyk-s) [reply commands/dev-deploy.js:88]: @ailisp The idea is to be able not to set `initFunction` in CLI when it has the default name (new).
But when we are setting the default value of `initFunction` to new, args become mandatory even when you are not initializing the contract and it breaks the logic.

**Pull request merged**